PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -81,8 +81,8 @@
 run2_miniAOD_80XLegacy.toModify(ptRatioRelForEle, srcLep = "slimmedElectronsUpdated")
 run2_nanoAOD_92X.toModify(ptRatioRelForEle, srcLep = "slimmedElectronsUpdated")
 
-from EgammaAnalysis.ElectronTools.calibratedElectronsRun2_cfi import calibratedPatElectrons
-calibratedPatElectrons.correctionFile = cms.string("PhysicsTools/NanoAOD/data/80X_ichepV1_2016_ele") # hack, should go somewhere in EgammaAnalysis
+from EgammaAnalysis.ElectronTools.calibratedPatElectronsRun2_cfi import calibratedPatElectrons
+calibratedPatElectrons.correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_17Nov2017_v1_ele_unc") 
 calibratedPatElectrons.semiDeterministic = cms.bool(True)
```

Comment:
is this going to resmear and correct again something that was smeared/corrected in miniAOD already?

Reply:
yes, we need to sort something about nano AOD. Currently I do not support in any way what nano AOD is doing and they should really have a meeting with E/gamma to discuss what they are doing.

Reply:
so, it sounds like this part is broken.
It should be possible now to figure out from a patElectron that it was smeared already.
Perhaps an exception or some similarly visible message can be added for an attempt to make another smearing.